pkg/bundle/ruleset/bundle.go

```diff
@@ -73,6 +72,16 @@ func FromBundle(b *bundlev1.Bundle) (*bundlev1.RuleSet, error) {
 			Constraints: []string{},
 		}
 
+		// Process the labels for each secret
+		for label, _ := range p.Labels {
+			r.Constraints = append(r.Constraints, fmt.Sprintf(`p.match_label(%q)`, label))
+		}
+
+		// Process the annotations for each secret
+		for annotation, _ := range p.Annotations {
```

Comment:
Same here, change it to: `for annotation := range p.Annotations {`

Reply:
Resolved via https://github.com/elastic/harp/pull/271/commits/5f6d466e892edae5bd080e184bd5b943c158e237